samigo/samigo-app/src/java/org/sakaiproject/tool/assessment/bundle/DeliveryMessages_de_DE.properties

```diff
@@ -7,7 +7,10 @@ add_q=Frage hinzuf\u00fcgen\:
 addr=Adressen\:
 admin=Admin
 done=Fertig
-ass_preview=Pr\u00fcfungsvorschau - Beispieldarstellung, wie Studierenden diese Pr\u00fcfung angezeigt wird                                                            
+exit_preview=Exit Preview
+ass_preview_1=Assessment Preview - This preview provides an approximate view of the layout of the assessment for students.
+ass_preview_2= The preview will <strong>not</strong> allow you to save or submit answers, display the timer if there is a time limit, or access any feedback.
+ass_preview_3= The preview also does <strong>not</strong> take into account students' ability (or lack thereof) to view images or other media embedded in questions.
```

Comment:
You don't need to add all these keys in each language. Just add them in the base properties file.